django_app/redbox_app/templates/base.html

```diff
@@ -20,7 +20,6 @@
 
   {% if environment | lower in ["dev", "preprod"] %}
     <script defer
-    event-environment="{{environment}}"
     data-domain="redbox-dev.ai.cabinetoffice.gov.uk" src="https://plausible.io/js/script.pageview-props.tagged-events.outbound-links.file-downloads.local.js"></script>
```

Comment:
Should this URL be hard coded?

Reply:
It's fine to be hard coded - it's built within the Plausible setup page where we copy and paste it :-)